applications/mne_analyze/mne_analyze/mainwindow.cpp

```diff
@@ -439,6 +442,8 @@ void MainWindow::createPluginViews(QSharedPointer<PluginManager> pPluginManager)
                 pWindow = m_pMultiView->addWidgetTop(pView, sCurPluginName);
             }
 
+            pWindow->setObjectName(sCurPluginName);
```

Comment:
maybe we can move this function call to addWidetBottom and addWidgetTop? This way others wouldd not forget to set it when using the MultiView.

Reply:
Yes. Turns out we had already done that and i did not notice. I'll remove this as it is redundant.